projects/igniteui-angular/src/lib/grids/grid-base.component.ts

```diff
@@ -4102,8 +4167,8 @@ export abstract class IgxGridBaseComponent extends DisplayDensityBase implements
                 sum += parseInt(col.width, 10);
             }
         }
-        if (this.showRowCheckboxes) {
-            sum += this.calcRowCheckboxWidth;
+        if (this.showRowCheckboxes || this.rowDrag) {
```

Comment:
sum += this.calcFixedWidth;
is just enough here without checking for showRowCehckboxec or rowDrag -> if both are false we will increase the width with 0